posts/2021-08-28-AndyMcCright_MeetTheTeam.adoc

```diff
@@ -63,49 +63,49 @@ All of these technologies are packaged as part of Open Liberty. For developers w
+Early in my career, I got some great advice (if a little buzzwordy...): build your brand; "be the ______ guy". A few years ago, my management team asked me to lead the JAX-RS team for IBM. I had to learn a lot about REST and how we implemented it in a short time. This got me involved in the Apache CXF project, where I've carved a small niche - mostly JAX-RS and later MicroProfile Rest Client. I'm starting to create a similar niche with RESTEasy. I've also become IBM's representative to the open source specification projects for Jakarta RESTful Web Services (the new name for JAX-RS) and MicroProfile Rest Client. In effect, I've become the "REST guy".
 
 image::/img/blog/AndyMcCright-2.png[Andy presenting about JAX-RS,width=70%,align="center"]
 
-My role has definitely changed from one that primarily fixed bugs to one that creates new features. My role has also added new facets like advocacy - I've been doing a lot more work-related social media (Twitter, LinkedIn, blogging) in concert with open source development. I've really enjoyed that aspect of my job. It's also now super-easy to create a new feature and then demo it on GitHub and blog about it with links to the source code. I think that really helps our users out - they know what our products can do and can see it working - and if they run into problems, they can talk directly with us over GH issues or StackOverflow, etc.
+My role has definitely changed from one that primarily fixed bugs to one that creates new features. My role has also added new facets, like advocacy. I've been doing a lot more work-related social media (Twitter, LinkedIn, blogging) in concert with open source development. I've really enjoyed that aspect of my job. It's also now super-easy to create a new feature and then demo it on GitHub and blog about it with links to the source code. I think that really helps our users out - they know what our products can do and can see it working - and if they run into problems, they can talk directly with us over GH issues or StackOverflow, etc.
 
 [#architect]
-=== What is involved in being a web services architect? What are your favourite parts of this role?
-More meetings... :-) Aside from the meetings, there is a lot of flexibility in my role. Some days, I'll work on product defects, while others I might help write documentation, and still others I might try to hype some new feature that our team recently delivered. It definitely involves working with a lot of people - and for a generally introverted person like me, it's strange but I kinda like that. 
+=== What is involved in being a web services architect? What are your favorite parts of this role?
```

Comment:
I'm all for American-izing this, but since Grace is British, I'd also be fine with the British spelling.  :)

Reply:
it's fine either way- I just changed it out of habit :)